### PR TITLE
Add new configuration for egress-router-cni

### PR DESCRIPTION
**Pull request by danielmellado**
This PR adds the proper configuration for openshift/egress-router-cni repo. It
updates the outdated, legacy one with a new one, in order to get the ART image
built.

### CODE REVIEW

File: ci-operator/config/openshift/egress-router-cni/openshift-egress-router-cni-master.yaml [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+  namespace: ocp
 tests:
-- as: unit
-  commands: make test-unit
-  container:
-    from: src
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
```
Comment (petr-muller) [inline ci-operator/config/openshift/egress-router-cni/openshift-egress-router-cni-master.yaml:28]: `openshift_installer` is the old, deprecated template-based tests. The tests should now use the newer template-based tests ([doc](https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration)) with the appropriate workflow (looks liks https://steps.ci.openshift.org/workflow/openshift-e2e-aws is the appropriate one - it runs the parallel [testsuite by default](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml#L10-L12))

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
/approve cancel 

(sorry, mistake, the review comment needs to be addressed)

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now 292323a44c466bb072c95d0cb23da7b508f4c149

**Comment by petr-muller** [comment]
/hold cancel

I think you just need to update the jobs (`make jobs`) and then this can merge.

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now ce7bdd662f1393414d66828f04aabe33b9fdaa95

**Review by petr-muller — APPROVED** [review]

**Pull request merged**